site/src/layouts/DetailBase/DetailBase.module.css

```diff
@@ -137,6 +137,25 @@
   box-shadow: none;
 }
 
+.headingContainer {
+  display: flex;
+  align-items: center;
+  gap: var(--salt-size-unit);
+  /*
+    Mosaic's still using Salt 1.1, so --salt-size-container-spacing is not available yet.
+
+    TODO: Replace line below with the commented-out one underneath once we upgrade Mosaic
+  */
```

Comment:
would eventually be replaced with --salt-spacing-300, size-container-spacing is deprecated